models/marts/tests/intermediate/int_model_test_summary.sql

```diff
@@ -13,7 +13,19 @@ count_column_tests as (
     select 
         relationships.direct_parent_id, 
         all_graph_resources.column_name,
-        count(distinct case when all_graph_resources.is_unique_test or all_graph_resources.is_not_null_test then relationships.resource_id else null end) primary_key_tests_count,
+        {%- for test_set in var('primary_key_test_macros') %}
+            {%- set outer_loop = loop -%}
+            {%- if test_set is iterable and (test_set is not string and test_set is not mapping) %}
```

Comment:
Same comment here... we're adding complexity by allowing such a wide range of acceptable inputs for the primary_key_test_macros variable.